lib/blob/cluster/cluster.go

```diff
@@ -347,7 +351,10 @@ func (c *cluster) WriteBLOB(data io.Reader) (*blob.Envelope, error) {
 		}
 	}
 
-	return nil, trace.Wrap(trace.NewAggregate(errors...), "not enough successfull writes")
+	if len(errors) == 0 {
+		return nil, trace.NotFound("not enough peers")
+	}
+	return nil, trace.Wrap(trace.NewAggregate(errors...), "not enough successful writes")
```

Comment:
Include how many successful writes we got in the message? I think in our case it will always be 0 though (since we set write factor to 1)?